Main/ClientApp/src/Actions/CollectionActions.ts

```diff
@@ -49,6 +49,40 @@ export async function loadSharedCollections() {
   });
 }
 
+export async function loadContributorsSharedCollections() {
+  let contributorsSharedCollections = await SharedCollectionsApi.getContributorsSharedCollections();
+  Dispatcher.dispatch({
+    actionType: ActionTypes.LOAD_CONTRIBUTORS_SHARED_COLLECTIONS,
+    payload: { contributorsSharedCollections: contributorsSharedCollections },
+  });
+}
+
 export async function shareCollection(shareData: SharedCollectionData) {
-  await SharedCollectionsApi.shareCollection(shareData);
+  let success = await SharedCollectionsApi.shareCollection(shareData);
+  if (success) {
+    loadCollections();
```

Comment:
Why do we want to load collections and sharedcollections here? Those things won't change on sharing

Reply:
true